plugins/generator-1.16.5/forge-1.16.5/templates/json/dimension/dimension_type.json.ftl

```diff
@@ -9,9 +9,9 @@
   "has_ceiling": ${data.worldGenType == "Nether like gen"},
   "coordinate_scale": 1,
   "ambient_light": <#if data.isDark>0<#else>0.5</#if>,
-  "logical_height": 256,
+  "logical_height": ${data.logicalHeight},
   "infiniburn": "minecraft:infiniburn_overworld",
-  "min_y": 0,
-  "height": 256,
+  "min_y": 0, <#-- Minecraft does not support custom heights in this version -->
+  "height": 256, <#-- Minecraft does not support custom heights in this version -->

```

Comment:
Comments here are totally odd, since you already indicate no support for these fields in this version with in-app tips and plugin definition files.